crates/astria-sequencer/src/ibc/ics20_transfer.rs

```diff
@@ -581,20 +580,27 @@ async fn execute_ics20_transfer_bridge_lock<S: StateWriteExt>(
     }
 
     // assert memo is valid
-    let deposit_memo: Ics20TransferDepositMemo =
+    let deposit_memo: memos::v1alpha1::Ics20TransferDeposit =
         serde_json::from_str(&memo).context("failed to parse memo as Ics20TransferDepositMemo")?;
 
     ensure!(
-        !deposit_memo.rollup_address.is_empty(),
+        !deposit_memo.rollup_deposit_address.is_empty(),
         "packet memo field must be set for bridge account recipient",
```

Comment:
nit: issue before this pr, but this error message is inaccurate

Reply:
I think it's correct, albeit obtuse: this check ensures that the address be non-empty.

Reply:
it doesn't validate that the memo field isn't empty though? It validates that the `rollup_desposit_address` is filled?

It used to be the case that we just used whatever was in the memo as the rollup address before we added shape. I don't think we updated the error message at that time.

Reply:
Ah, I see what you mean now. I didn't read the text carefully enough. Updated in  b83d6c59